src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -120,7 +120,7 @@ class NetworkOPsImp final
     {
         struct Counters
         {
-            std::uint64_t transitions = 0;
+            std::uint32_t transitions = 0;
```

Comment:
Did you intend to change this back to `uint32_t`?  If so, then I think you could skip the uses of `std::to_string()` below.  But I don't believe that's your intent.

Reply:
Yes, I intended to revert and I did revert the conversion for transitions. And to_string() was already in place for the dur member, so that's unchanged.

Reply:
Thanks Mark.  It looks like the added `std::to_string()` calls deal with your removal of the 64-bit JSON interface.  Sorry I didn't catch that before.